airbyte-cdk/python/airbyte_cdk/sources/file_based/config/file_based_stream_config.py

```diff
@@ -56,25 +64,6 @@ def validate_file_type(cls, v: str) -> str:
             raise ValueError(f"Format filetype {v} is not a supported file type")
         return v
 
-    @validator("format", pre=True)
-    def transform_format(cls, v: Mapping[str, str]) -> Any:
```

Comment:
at this point since we are back to only supporting one format and not the map, there is no distinction between the shape of the config that requires transforming (although the format fields can differ)